tools/alpine/versions.aarch64

```diff
@@ -87,14 +87,14 @@ libattr-2.4.47-r6
 libblkid-2.28.2-r2
 libburn-1.4.6-r0
 libbz2-1.0.6-r5
+libc-dev-0.7.1-r0
+libc-utils-0.7.1-r0
```

Comment:
yes, i tried that a while back as well and it didn't fix it. forgot to mention it...